src/containers/sidebars/featured-place-card/featured-place-card-component.jsx

```diff
@@ -92,13 +94,37 @@ function FeaturedPlaceCardComponent({
                     alt={featuredPlace.title}
                   />
                 )}
+                {featuredPlace.link && (
+                  <a
```

Comment:
It will be nice to use the Link component that we already have. Using these components allows easier changes in style and functionality on the whole page. Also helping with maintainability.